src/showcase/app/public/public-examples/tag-examples/tag-examples.module.ts

```diff
@@ -8,12 +8,23 @@ import { SbbTagModule } from '@sbb-esta/angular-public/tag';
 
 import { provideExamples } from '../../../shared/example-provider';
 
-import { TagExampleComponent } from './tag-example/tag-example.component';
+import { TagAdvancedExample } from './tag-advanced-example/tag-advanced-example.component';
+import { TagLinkExample } from './tag-link-example/tag-link-example.component';
+import { TagReactiveFormsExample } from './tag-reactive-forms-example/tag-reactive-forms-example.component';
+import { TagTemplateFormsExampleComponent } from './tag-template-forms-example/tag-template-forms-example.component';
```

Comment:
In https://github.com/sbb-design-systems/sbb-angular/blob/master/MERGE_MIGRATION.md at step 1:
Rename files with *.component.* pattern to *.* and update references (e.g. autocomplete.component.ts should become autocomplete.ts
Are examples excludet from this rule?

Reply:
these are the old examples which can be left as they are, but I see that they don't have consistent Component suffix. I changed it :-)